src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,11 @@
-const booleanToWord = boolean => {};
+const booleanToWord = boolean => {
+    const a = boolean.toString();
+    if (a == 'true')
+    return 'Yes';
+
+    if (a == 'false')
+    return 'No';
+    
+};
```

Comment:
I know there are a lost of suggestions on this, but it's still a correct and working answer, so don't be disheartened. This code achieves all the goals we asked. :)